microprofile.ls/org.eclipse.lsp4mp.ls/src/main/java/org/eclipse/lsp4mp/services/properties/PropertiesFileCompletions.java

```diff
@@ -229,6 +229,19 @@ private static void collectPropertyKeySuggestions(int offset, Node node, Propert
 				insertText.append(' ');
 			}
 
+			// replace with current property value if exist

```

Comment:
I guess this question is directed more at @angelozerr , but if `Assign` inherited from `BasePropertyValue`, then the 2 cases could be combined. Is there a reason it doesn't ? I guess an assignment node need not have a value ?

Reply:
We can live with this for now.